test/nodes/test_connector.py

```diff
@@ -1,11 +1,14 @@
 from typing import List
 
 import json
+import time
 from pathlib import Path
 
 import pytest
 from selenium.webdriver.common.by import By
 
+from selenium.webdriver.common.by import By
```

Comment:
Just a duplicate import

Reply:
I also removed the time unused import.

Thank you!